template/web/wp-content/themes/<%= slug %>/app/Managers/AssetsManager.php

```diff
@@ -18,6 +18,6 @@ class AssetsManager implements ManagerInterface {
 	 * @inheritDoc
 	 */
 	public function run() {
-		new Assets( get_template_directory() );
+		return new Assets( sprintf( '%s/config/', get_template_directory() ) );
```

Comment:
**issue:** This change is not tied to the TransientManager feature, it should be made in a different PR

Reply:
See #61